merlin-driver/src/main/java/gov/nasa/jpl/aerie/merlin/driver/MissionModelBuilder.java

```diff
@@ -63,7 +63,7 @@ public void resource(final String name, final Resource<?> resource) {
   }
 
   @Override
-  public String daemon(final TaskFactory task) {
+  public <Return> String daemon(final TaskFactory<Return> task) {
```

Comment:
You can use a wildcard here, too.

```suggestion
  public String daemon(final TaskFactory<?> task) {
```

Reply:
I'll make a ticket for revisiting daemon tasks, particularly to replace the "lump into one parent task" strategy